src/librustc_resolve/lib.rs

```diff
@@ -2677,8 +2665,7 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
                                    path: &Path,
                                    namespace: Namespace,
                                    check_ribs: bool)
-                                   -> AssocItemResolveResult
-    {
+                                   -> AssocItemResolveResult {
```

Comment:
I actually prefer the linebreak before the { too, and have built in an option to do it in rustfmt. I feel like I should make an RFC for the change to the style guide or something though.